### PR TITLE
{devel}[intel/2017a] GObject-Introspection v1.53.5, GObject-Introspection v1.53.5

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
requires #5037, #5040

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2066.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/607285da1dbd0dc6575aef77b0757fe6 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2572.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/8ec051b38bb33e7e13ce2ee02c5c6217 for a full test report.

File: easybuild/easyconfigs/g/GObject-Introspection/GObject-Introspection-1.53.5-intel-2017a-Python-2.7.13.eb [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+buildopts = "PYTHON=python"
+
+modextrapaths = {
+    'GI_TYPELIB_PATH': 'share',
+    'XDG_DATA_DIRS': 'share',
+}
+
+sanity_check_paths = {
+    'files': ['bin/g-ir-%s' % x for x in ['annotation-tool', 'compiler', 'generate', 'scanner']] +
+             ['lib/libgirepository-1.0.%s' % x for x in ['so', 'a']],
```
Comment (verdurin) [inline easybuild/easyconfigs/g/GObject-Introspection/GObject-Introspection-1.53.5-intel-2017a-Python-2.7.13.eb]: @boegel Another `SHLIB_EXT` case here

File: easybuild/easyconfigs/g/GObject-Introspection/GObject-Introspection-1.53.5-intel-2017a-Python-3.6.1.eb [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+buildopts = "PYTHON=python3"
+
+modextrapaths = {
+    'GI_TYPELIB_PATH': 'share',
+    'XDG_DATA_DIRS': 'share',
+}
+
+sanity_check_paths = {
+    'files': ['bin/g-ir-%s' % x for x in ['annotation-tool', 'compiler', 'generate', 'scanner']] +
+             ['lib/libgirepository-1.0.%s' % x for x in ['so', 'a']],
```
Comment (verdurin) [inline easybuild/easyconfigs/g/GObject-Introspection/GObject-Introspection-1.53.5-intel-2017a-Python-3.6.1.eb]: `SHLIB_EXT` again

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 4 out of 4 (2 easyconfigs in this PR)
ca005.camp.thecrick.org - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2640 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/ff910a325a6ac1557638c8d0a8523bab for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2417.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/a130e64468c37a99f184e764b65488fe for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2071.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/7ca82af2541ce9c6f54bef91f121abd1 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
ca005.camp.thecrick.org - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2640 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/6f0a0e38615bc1745a2969a9571ecf5a for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
ca003.camp.thecrick.org - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2640 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/6d39513262b4fc245f7214f5dba96f87 for a full test report.

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Pull request merged**